modules/openapi-generator/src/main/resources/typescript-fetch/modelEnum.mustache

```diff
@@ -4,6 +4,8 @@ export function {{classname}}FromJSON(json: any): {{classname}} {
     return {{classname}}FromJSONTyped(json, false);
 }
 
+// ignoreDiscriminator not used
+// @ts-ignore
```

Comment:
**question:** I don't really get it why the ignoreDiscriminator parameter is declared here at all.
Would it be appropriate to just remove it instead of suppressing an error?

Reply:
No, it would be not appropriate.

Here it is easy recognizable that the ``imageDiscriminator`` is unused. But it's hard to recognize that in the calling class.